crates/bevy_ecs/src/bundle.rs

```diff
@@ -8,13 +8,33 @@ use crate::{
 use bevy_ecs_macros::all_tuples;
 use std::{any::TypeId, collections::HashMap};
 
-/// An ordered collection of components
-///
-/// See [Bundle]
-/// # Safety
-/// [Bundle::type_info] must return the TypeInfo for each component type in the bundle, in the _exact_
-/// order that [Bundle::get_components] is called.
-/// [Bundle::from_components] must call `func` exactly once for each [TypeInfo] returned by [Bundle::type_info]
+/** An ordered collection of components, commonly used for spawning entities, and adding and removing components in bulk.
+
+You cannot query for a bundle, only individual components within it.
```

Comment:
Haha while we're touching this file, can you also add a note on "tuple bundles"?